src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -39,7 +40,7 @@ class QuestionHelper extends Helper
      * @param OutputInterface $output   An OutputInterface instance
      * @param Question        $question The question to ask
      *
-     * @return string The user answer
+     * @return string|array|mixed The user answer
```

Comment:
I agree :+1: BTW it's not clear if `default` should be a `string` or `mixed`. In Question it's `mixed` but in SymfonyStyle it's `string|null`...